src/main/java/com/rentus/RentusApplication.java

```diff
@@ -2,30 +2,41 @@
 
 import com.rentus.controllers.*;
 import com.rentus.utility.SessionFactory;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+import spark.Spark;
 
+import java.util.HashMap;
 import java.util.logging.Logger;
 
+import static spark.Spark.after;
 import static spark.Spark.port;
 import static spark.Spark.staticFiles;
 
 public class RentusApplication {
 	private final static Logger LOGGER = Logger.getLogger(RentusApplication.class.getName());
 	public static void main(String[] args) {
 		try{
+
 			port(1234);
 			//staticFileLocation("dist/index.html");
 
+
+
+
 			String projectDir = System.getProperty("user.dir");
 			String staticDir = "/src/main/resources/public";
 			staticFiles.externalLocation(projectDir + staticDir);
 
+			RentusApplication.apply();
 			SessionFactory.getInstance();
-			ToolsController.addtool();
```

Comment:
why removed?

Reply:
I renamed Tools to use the name Products. and blah blah